client/dogs/SearchPage.js

```diff
@@ -36,16 +36,16 @@ export default class IndexPage extends React.Component {
     const { query } = ctx;
 
     let results = null;
-    //console.log('QUERY', query);
 
+    //THIS SEARCH FUNCTION IS NOT BEING USED NOW
```

Comment:
Ooops! The SHOUTY CAPITALS were supposed to remind me to delete it after I resolved errors :P Done